core/arch/decode.h

```diff
@@ -310,15 +310,19 @@ enum {
     OPSZ_100, /**< 100 bytes. Needed for load/store of register lists. */
     OPSZ_104, /**< 104 bytes. Needed for load/store of register lists. */
     /* OPSZ_108 already exists */
-    OPSZ_112,             /**< 112 bytes. Needed for load/store of register lists. */
-    OPSZ_116,             /**< 116 bytes. Needed for load/store of register lists. */
-    OPSZ_120,             /**< 120 bytes. Needed for load/store of register lists. */
-    OPSZ_124,             /**< 124 bytes. Needed for load/store of register lists. */
-    OPSZ_128,             /**< 128 bytes. Needed for load/store of register lists. */
-    OPSZ_SCALABLE,        /** Scalable size for SVE vector registers. */
-    OPSZ_SCALABLE_PRED,   /** Scalable size for SVE predicate registers. */
-    OPSZ_16_vex32_evex64, /**< 16, 32, or 64 bytes depending on EVEX.L and EVEX.LL'. */
-    OPSZ_vex32_evex64,    /**< 32 or 64 bytes depending on EVEX.L and EVEX.LL'. */
+    OPSZ_112,                /**< 112 bytes. Needed for load/store of register lists. */
+    OPSZ_116,                /**< 116 bytes. Needed for load/store of register lists. */
+    OPSZ_120,                /**< 120 bytes. Needed for load/store of register lists. */
+    OPSZ_124,                /**< 124 bytes. Needed for load/store of register lists. */
+    OPSZ_128,                /**< 128 bytes. Needed for load/store of register lists. */
+    OPSZ_SCALABLE,           /** Scalable size for SVE vector registers. */
+    OPSZ_SCALABLE_PRED,      /** Scalable size for SVE predicate registers. */
+    OPSZ_16_vex32_evex64,    /**< 16, 32, or 64 bytes depending on EVEX.L and EVEX.LL'. */
+    OPSZ_vex32_evex64,       /**< 32 or 64 bytes depending on EVEX.L and EVEX.LL'. */
+    OPSZ_16_of_vex32_evex64, /**< 128 bits: half of YMM or quarter of ZMM depending on
+                              * EVEX.LL'.
+                              */
+    OPSZ_32_of_evex64,       /**< 256 bits: half of ZMM. */
```

Comment:
Same thing: this needs a base size.  If it's always out of 64 bytes, it shouldn't have the evex prefix.

Reply:
Done.